garnett/templates/garnett/language_selector.html

```diff
@@ -11,7 +11,7 @@
         {% else %}
             {% if selector == "cookie" %}
                 <button title="{{ lang }}"
-                    onclick='document.cookie = "GARNETT_LANGUAGE_CODE={{lang}}"; location.reload(); return false'
+                    onclick='document.cookie = "GARNETT_LANGUAGE_CODE={{lang.language}};path=/"; location.reload(); return false'
```

Comment:
Why did we need to add the path / here - I'm a little bit spooked by that in terms of crafting URLs